invenio/modules/accounts/templates/accounts/settings/helpers_base.html

```diff
@@ -22,6 +22,7 @@
 {%- block panel_start scoped %}
 <div class="panel-group panel-bot-margin"{% if id %} id="{{id}}"{% endif %}>
     <div class="panel panel-default">
+        {%- block panel_wrapper scoped %}
```

Comment:
what about calling it consistently `panel_heading` like you are doing in https://github.com/inveniosoftware/invenio/pull/2963/files#diff-b66b7d32b226b41b656198b33c89e2caR25